app/app-info.js

```diff
@@ -0,0 +1,90 @@
+'use strict'
+
+const fs = require('fs')
+const path = require('path')
+
+const gitInfo = require('./git-info')
+const distInfo = require('../script/dist-info')
+
+const projectRoot = path.join(__dirname, '..')
+
+const devClientId = '3a723b10ac5575cc5bb9'
+const devClientSecret = '22c34d87789a365981ed921352a7b9a8c3f69d54'
+
+const channel = distInfo.getReleaseChannel()
+
+function getCLICommands() {
+  return (
+    // eslint-disable-next-line no-sync
```

Comment:
This has to be moved down one line — `eslint-disable-next-line` only disables for the next line.

Reply:
I think this is right, as moving it down reports an error in VSCode:

<img width="599" src="https://user-images.githubusercontent.com/359239/32635596-1c7a981c-c605-11e7-81dd-c4188ab4868b.png">

I've aded these new files to eslint in bfe82cf3d805b542e89d6ac104362d662cb47210 just in case.